openapi_to_fastapi/model_generator.py

```diff
@@ -23,15 +23,15 @@ def generate_model_from_schema(schema: str, format_code: bool = False) -> str:
     parser = OpenAPIParser(
         source=schema,
         data_model_type=pydantic_model.BaseModel,
-        data_model_root_type=pydantic_model.CustomRootType,
+        data_model_root_type=pydantic_model.RootModel,
         data_type_manager_type=pydantic_model.DataTypeManager,
         data_model_field_type=pydantic_model.DataModelField,
         base_class="pydantic.BaseModel",
         custom_template_dir=None,
         extra_template_data=None,
-        target_python_version=PythonVersion.PY_37,
+        target_python_version=PythonVersion.PY_38,
         dump_resolve_reference_action=None,
-        validation=True,
+        validation=False,
```

Comment:
I think it'd be nice to add a comment about why it's disabled, something like

```suggestion
        # Validation of OpenAPI 3.1 specs isn't supported right now. Issues: 
        # https://github.com/koxudaxi/datamodel-code-generator/issues/1649 and 
        # https://github.com/RonnyPfannschmidt/prance/issues/95
        validation=False,
```